client/my-sites/comments/comment-list/index.jsx

```diff
@@ -88,21 +89,28 @@ export class CommentList extends Component {
 			this.setState( {
 				isBulkEdit: false,
 				lastUndo: null,
-				page: 1,
 				persistedComments: [],
 				selectedComments: [],
 			} );
 		}
 	}
 
-	changePage = page => {
-		const total = Math.ceil( ( this.props.comments.length + this.state.persistedComments.length ) / COMMENTS_PER_PAGE );
-		this.props.recordChangePage( page, total );
+	changePage = pageNumber => {
+		const {
+			recordChangePage,
+			siteFragment,
+			status,
+		} = this.props;
 
-		this.setState( {
-			page,
-			selectedComments: [],
-		} );
+		recordChangePage( page, this.getTotalPages() );
```

Comment:
This should be `pageNumber` no?

Reply:
yep, it should be `recordChangePage( pageNumber, this.getTortalPages() )`. Updated on 230d5fc6f45ab0e749c8c1b2474e2a1fc63a1329.

Reply:
Is this worth having? What would we use these stats for? I'd rather cut out the code.

Reply:
... _track all the things!?_  ಠ_ಠ